### ISSUE TITLE
Feature request - auto favorite filters

### ISSUE THREAD

**Issue by ajellio** [Mishiranu/Dashchan#300]
Hi,

I like your app a lot, thanks for your good work. One feature I miss from Kuroba is the ability to use filters (just as in the existing "Autohide" feature) to automatically favorite threads. For example, a filter searching "dashchan" on /g/ to add those threads to my favorites automatically.

Use case: there are a few subjects I like to follow on many boards (but don't care about the board as a whole) and without this feature I have to go to each of those boards, go to search feature and search my keywords manually to find and watch those threads.

Thanks for the app and I hope you consider this feature

**Comment by Mishiranu** [comment]
Is this a "filter watching"? I heard these words before, but didn't have any ideas what do they mean.

How does it works? Do you need to open the board manually to check for new threads, or it should work in background somehow?

**Comment by ajellio** [comment]
Yes in the background if possible so you don't have to switch to each board you have filters set up for. I can see this would be quite a lot of activity in the background so I'm not sure how Kuroba does this but it's a very good feature.

**Comment by darlopvil** [comment]
It would be awesome if it could be implemented
